src/middleware/addUserContext.ts

```diff
@@ -16,12 +16,12 @@ const addUserContext = (
 	res: Response,
 	next: NextFunction,
 ): void => {
-	const authenticationId = req.auth?.sub;
-	if (authenticationId === undefined || authenticationId === '') {
+	const keycloakUserId = req.auth?.sub;
+	if (keycloakUserId === undefined || keycloakUserId === '') {
```

Comment:
Should we also defend against the system user ID here? It's not completely clear to me if the intent of `authenticationId === ''` was to check for a missing ID or for an incorrectly used system ID or both.

Reply:
It was indeed, and in fact... I really don't want hard coded values (because it leads to mistakes like this).  I wonder if there would be a good way to load that user ID directly from the DB on service start, and to check against somehow...  I'll figure something good out.

Reply:
I've added a new `config.ts` which is initialized on server start and which populates a `systemUser` which can be accessed via `getSystemUser()` and I think this will do the trick.